vibecoding-toolkit/server/tools/agent.ts

```
@@ -0,0 +1,354 @@
+        params.push(simpleExpr.value);
+        return { clause: `${fieldName} >= $${paramIndex}`, params };
+      case "lt":
+        params.push(simpleExpr.value);
+        return { clause: `${fieldName} < $${paramIndex}`, params };
+      case "lte":
+        params.push(simpleExpr.value);
+        return { clause: `${fieldName} <= $${paramIndex}`, params };
+      case "in":
+        const values = Array.isArray(simpleExpr.value)
+          ? simpleExpr.value
+          : [simpleExpr.value];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)